project/src/main/webapp/profile.js

```diff
@@ -46,10 +46,17 @@ async function updateUserEventsParticipating() {
 async function updateUserEventsVolunteering() {
   const userEventsVolunteering = await getUserEvents('volunteering');
   for (const eventVolunteering of userEventsVolunteering) {
-    populateEventContainer(eventVolunteering, 'events-volunteering');
+    const event = eventVolunteering.event;
+    await populateEventContainer(event, 'events-volunteering');
+    addVolunteerRole(event.eventId, eventVolunteering.opportunityName);
   }
 }
 
+function addVolunteerRole(eventId, opportunity) {
```

Comment:
Add javadoc, I believe linting gives an error here otherwise?

Reply:
Although the linting gives an error for some lack of JSDoc, I feel like it would produce a lot of redundancy if I add docs to functions like `addVolunteerRole` which seems like a worse idea than just a descriptive function name